### PR TITLE
New Jungle Ruin: Abandoned Library

### PR DESCRIPTION
**Pull request by meemofcourse**
## About The Pull Request

library2

(ignore the branch name i was working on that originally)

A library, raided after the owners refused to budge under raiders' demands, and long abandoned. Maybe there's something good still in there?

Threats:
1x Gorilla
7x Monkey

Main Loot:
-Book of Babel
-Singular-use Codewords Manual (a bit redundant if someone has a book of babel now that i think about it)
-Bible Gun (sketchin) with 2x Magazines
-Curator Whip and Outfits (+ Book Uploading i suppose)

Other Loot:
-Someone's lunch (chicken nuggets, grilled cheese, cola and an apple)
-Paperwork stuff
-Three pizzas and a beer holder near the DnD nerds
-Four action figures
-Books
-Cap Gun 
-Replica Katana
-Uncharged baton
-More books
-Card Set for CAS
-Mech Drill ("broken thermal drill" near the safe)
-You won't believe this, but it's more books

## Why It's Good For The Game
more ruin good

## Changelog

:cl:
add: Abandoned library
/:cl:

### CODE REVIEW

**Comment by Xylarr** [comment]
the display key from the safe is not unlocking the babel's case

**Comment by meemofcourse** [comment]
i looked at the code and apparently it only does opens the cases when they've been there since roundstart. seems like you're going to have to break it!

**Comment by AverageUser67** [comment]
Are they the angry monkey subtype

**Comment by meemofcourse** [comment]
yes, they have a 10% chance to spawn with the justice helmet

File: _maps/map_catalogue.txt [hunk 1]
```diff
@@ -133,6 +133,9 @@ Find the key for using this catalogue in "map_catalogue_key.txt"
 		Size = (x = 100)(y = 75)(z = 1_)
 		Tags = "Medium Combat Challenge", "Major Loot", "Hazardous", "Liveable"
 
+		File Name "_maps\RandomRuins\JungleRuins\jungle_abandoned_library
+		Size = (x = 36)(y = 35)(z = 1_)
```
Comment (Latentish) [inline _maps/map_catalogue.txt]: minor typo on the z count

Comment (meemofcourse) [reply _maps/map_catalogue.txt]: piss

Comment (Ms-Mee) [reply _maps/map_catalogue.txt]: piz_

**Comment by Latentish** [comment]
Gonna tm this next uptime if possible.

**Review by Latentish — DISMISSED** [review]
You're gonna need to fix wallmounts now that anywalls are a thing.
image

What is his name.
image

The monkeys do not attack you.

**Comment by meemofcourse** [comment]
> What is his name.

smart moth
> The monkeys do not attack you.

one would think the aggressive subtype of a mob would make it attack you. i'll look into it

**Comment by Latentish** [comment]
Fix the monkeys issue and we can gamer that bread.

**Comment by Latentish** [comment]
Please address the funny monkey issue so we can merge this? Or use more than one gorilla.

**Comment by Latentish** [comment]
Please draft, close, or update your pr.

**Comment by meemofcourse** [comment]
waiter more gorillas please

**Review by Erikafox — APPROVED** [review]

**Pull request merged**